spec.md

```diff
@@ -92,39 +91,52 @@ The `Span` interface must have the following capabilities:
                 +--------+--------+     CHILD OF SPAN C), AS WELL
                 |        |        |     AS SPANS G, H, AND I.
             [Span G] [Span H] [Span I]
-```
+~~~
 
 
-### Tracer
+## Tracer
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
-- Provide some form of access to a SpanPropagator (whether that's through embedding, inheritance, containment, or something else is per-platform)
+- Start a new `Span`. There must be a way for the caller to specify a parent `Span`, an explicit start timestamp (other than "now"), and an initial set of `Span` tags. **(py: `start_span`, go: `StartSpanWithOptions`)**
+- Return an `Injector` instance given a format identifier, or nil/null if the format is not supported by the implementation (see [Injector and Extractor](#injector-and-extractor))
+- Return an `Extractor` instance given a format identifier, or nil/null if the format is not supported by the implementation (see [Injector and Extractor](#injector-and-extractor))
 
-### SpanPropagator
+A note about **"format identifiers"** per the `Injector` and `Extractor` lookup methods above: the nature of the "format identifier" may vary from platform to platform, but in all cases they should be drawn from a global namespace. New formats must not *require* changes to the core OpenTracing platform APIs, though those core platform APIs must define a few basic/general formats (like string maps and binary blobs). For example, if the maintainer of EsotericRPCFramework wanted to define an EsotericRPCFramework injection and extraction format, she or he must be able to do so without sending a PR to OpenTracing maintainers (though of course OpenTracing implementations are not required to suppor the EsotericRPCFramework format).
```

Comment:
This really needs examples. for example nested in here is "few basic/general formats (like string maps and binary blobs)", but we don't give an example of what a format identifier might look like. pseudocode might help clear this up.